src/components/ColorPicker/ColorPickerItem/ColorPickerItem.tsx

```diff
@@ -32,11 +32,7 @@ export class ColorPickerItem extends React.Component<
 
     return (
       <RadioButton
-        {...styles(
-          'root',
-          { checked, focused: this.state.focused },
-          { ...this.props },
-        )}
+        className={st(classes.root, { checked, focused: this.state.focused })}
```

Comment:
```suggestion
        className={st(classes.root, { checked, focused: this.state.focused }, this.props.className)}
```

`className` is defined in `ColorPickerItemProps`, so should be passed

Reply:
good catch